examples/load-data-and-cgm-stats.r

```diff
@@ -0,0 +1,120 @@
+####################################################################################
+# description: Example for loading Tidepool data and calculating cgm stats in R
+# version: 0.0.1
+# created: 2018-07-17
+# author: Jason Meno
+# license: BSD-2-Clause
+####################################################################################
+
+# Set location of data
+dataPath = "../example-data/"
```

Comment:
please use something like file.path() so we can avoid using slashes in the file path. For example, dataPath = file.path("..", "example-data")

Reply:
Fixed, now using file.path()